docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -1,17 +1,2133 @@
+-------------
+
+*Returns:* `image`
+
+
+[float]
+[[location_fn]]
+=== `location`
+
+Uses the browser's location functionality to get your current location. Usually 
+quite slow, but fairly accurate.
+
+*Accepts:* `null`
+
+*Returns:* `datatable`
+
+
+[float]
+[[l_fns]]
+== L
+[[lt_fn]]
+=== `lt`
+
+Returns whether the _context_ is less than the argument passed in.
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `value`
+|`number`, `string`
+|The value to compare the _context_ to
+|===
+
+*Returns:* `boolean`
+
+
+[float]
+[[lte_fn]]
+=== `lte`
+
+Returns whether the _context_ is less than or equal to the argument passed in.
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `value`
+|`number`, `string`
+|The value to compare the _context_ to
+|===
+
+*Returns:* `boolean`
+
+
+[float]
+[[m_fns]]
+== M
+[[mapColumn_fn]]
+=== `mapColumn`
+
+Adds a column calculated as the result of other columns, or not. See also <<alterColumn_fn>> and <<staticColumn_fn>>.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `column`
+|`string`
+|The name of the resulting column
+
+|`expression` ***
+
+Alias: `exp`, `fn`, `function`
+|`boolean`, `number`, `string`, `null`
+|A Canvas expression that will be passed to each row as a single row `datatable`
+|===
+
+*Returns:* `datatable`
+
+
+[float]
+[[markdown_fn]]
+=== `markdown`
+
+Adds an element for rendering Markdown text. Great for single numbers, metrics, or paragraphs of text.
+
+*Accepts:* `datatable`, `null`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *†*
+
+Alias: `expression`, `content`
+|`string`
+|A `string` containing Markdown. You can pass this multiple 
+times to achieve concatenation 
+
+Default: `""`
+
+|`font`
+|`style`
+|Font settings. Technically, you can put other styles in here too 
+
+Default: `{<<font_fn>>}`
+|===
+
+*Returns:* `render`
+
+
+[float]
+[[math_fn]]
+=== `math`
+
+Interprets a math expression, with a `number` or `datatable` as context. Datatable 
+columns are available by their column name. If you pass in a `number`, it is 
+available as `"value"` (without the quotes).
+
+*Accepts:* `number`, `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `expression`
+|`string`
+|An evaluated TinyMath expression, see <<canvas-tinymath-functions>>
```

Comment:
This should be two sentences:

`An evaluated TinyMath expression. See <<canvas-tinymath-functions>>